settings/Localization.setting.php

```diff
@@ -409,6 +409,28 @@
       'callback' => 'CRM_Core_I18n::languages',
     ],
   ],
+  'format_locale' => [
+    'group_name' => 'Localization Preferences',
+    'group' => 'localization',
+    'name' => 'format_locale',
+    'type' => 'String',
+    'quick_form_type' => 'Select',
+    'html_type' => 'Select',
+    'html_attributes' => [
+      'multiple' => 0,
```

Comment:
There's something wrong here it seems to let me select multiple, although it only saves one. But more importantly I can't clear the field value once it's set.

Reply:
Yeah, that's weird. Simply undeclaring`multiple=>0` brings it back to a proper single-select, but then it doesn't display the default/null value nicely. Added 9d14b6f716d5db15fce70a1648c6c206f3557047 to address those two things together.